### PR TITLE
add README

### PR DESCRIPTION
**Pull request by vdice**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+Want to work on Porter with us? 💖 We are actively seeking out new contributors
+with the hopes of building up both casual contributors and enticing some of you
+into becoming reviewers and maintainers.
+
+<p align="center">Start with our <a href="https://porter.sh/contribute/">New Contributors Guide</a>
+
+Porter wouldn't be possible without our [contributors][contributors], carrying
+the load and making it better every day! 🙇‍♀️
+
+[contributors]: /CONTRIBUTORS.md
```
Comment (carolynvs) [inline README.md]: This needs to link to the porter repo for now

**Review by carolynvs — COMMENTED** [review]
just the bad link that needs to be fixed then it's GTG 🚀

File: README.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+Want to work on Porter with us? 💖 We are actively seeking out new contributors
+with the hopes of building up both casual contributors and enticing some of you
+into becoming reviewers and maintainers.
+
+<p align="center">Start with our <a href="https://porter.sh/contribute/">New Contributors Guide</a>
+
+Porter wouldn't be possible without our [contributors][contributors], carrying
+the load and making it better every day! 🙇‍♀️
+
+[contributors]: https://porter.sh/src/CONTRIBUTING.md
```
Comment (carolynvs) [inline README.md]: oops! Sorry this link should be this 

```suggestion
[contributors]: https://porter.sh/src/CONTRIBUTORS.md
```

**Review by carolynvs — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (vdice) [reply README.md]: Oh yea :)

**Review by carolynvs — APPROVED** [review]
Thanks for making this repo! 🙇‍♀️

**Pull request merged**